app/controllers/application_controller.rb

```diff
@@ -1,9 +1,14 @@
 class ApplicationController < ActionController::Base
   include AnalyticsHelper
-  
+
   protect_from_forgery with: :exception
   before_action :set_visitor_id
   before_action :configure_permitted_parameters, if: :devise_controller?
+  before_filter :set_phone_number
+
+  def set_phone_number
+    @user_phone_number = PhoneNumberParser.format_for_display(ENV['TWILIO_PHONE_NUMBER'])
```

Comment:
1. Can we call it `@clientcomm_phone_number` instead? Logically it's not the user's number as all the users share the same number.
1. Can we use `||=` instead of `=`, like in `set_visitor_id`? So we're not re-parsing the number every time.